EDU-14349: Release note - 'FastStore: New Global Content Types' (#1756)
vtexdocs/dev-portal-content: docs/faststore/docs/headless-cms/overview.mdx

> ![CMS Settings](https://vtexhelp.vtexassets.com/assets/docs/src/headless-cms-projects___5472a0870899c945ece289905ac92758.png)

Reviewer: Maybe we could update this image showing the default content types

Content Types are the building blocks for creating a scalable and maintainable storefront in FastStore. After onboarding via the [FastStore WebOps](https://developers.vtex.com/docs/guides/faststore/1-onboarding-overview) app, these Content Types are pre-implemented in your starter store, enabling you to quickly build pages like the Home, Product Listing, and Custom Landing.

> Content Types are organized by [Scopes](https://developers.vtex.com/docs/guides/faststore/headless-cms-managing-content-with-scopes-and-requiredscopes#scopes), which define the context in which each Content Type can be used. They categorize Content Types based on their functionality. For example, a Content Type with a `pdp` scope can only be used on PDPs.

Reviewer 2: ```suggestion
Content Types are categorized by [Scopes](https://developers.vtex.com/docs/guides/faststore/headless-cms-managing-content-with-scopes-and-requiredscopes#scopes), which define the context where they can be used. For example, a Content Type with a `pdp` scope can only be used on PDPs.
```

| Global Sections | `globalSections` | `global` | Defines reusable sections that appear across pages. |
| Global Header Sections | `globalHeaderSections` | `global` | Defines the content for the global header, displayed at the top of every page. |
| Global Footer Sections | `globalFooterSections` | `global` | Defines the content for the global footer, displayed at the bottom of every page. |
| Landing Page | `landingPage` | `landing` and `custom` | Creates custom landing pages with SEO optimization. |
| Home | `home` | `home` | Defines the structure and SEO metadata for the home page. |
| Product Page | `pdp` | `pdp` | Defines the structure and template for Product Detail Pages (PDP). |
| Product List Page | `plp` | `plp` | Defines the structure and settings for Product Listing Pages (PLP). |
| Search Page | `search` | `plp` and `search` | Defines the structure and settings for search results pages. |
| Login | `login` | - | Defines the structure for the login page. |
| Error 500 | `500` | - | Defines the structure for the 500 Internal Server Error page. |
| Error 400 | `400` | - | Defines the structure for the 400 pages, including the 404 Not Found Error page. |

### Sections

> For each Content Type, different sections will be available to build a page. Sections define the content structure of a React component, such as the [Alert](https://developers.vtex.com/docs/guides/faststore/molecules-alert).

Reviewer 2: ```suggestion
Each Content Type has different Sections available to build a page. Sections represent key React components, such as Hero or Product Gallery, manageable within the CMS. Editors can visually configure and arrange storefront Sections without needing to code. For example, the [Alert component](https://developers.vtex.com/docs/guides/faststore/molecules-alert) is a FastStore UI component and Section that can be used in various Content Types.
```

![CMS Section](https://vtexhelp.vtexassets.com/assets/docs/src/cms-sections___ccc8b00d73f222d249a009eb195c8e0c.png)

> Sections can be reused on different pages of an ecommerce. You have the flexibility to choose which sections will be available at the CMS. Check the following example of the [Hero Section](https://developers.vtex.com/docs/guides/faststore/organisms-hero) being used:

Reviewer 2: ```suggestion
Sections are reusable across different pages. Developers determine which sections are available in the CMS. Below is an example of the [Hero Section](https://developers.vtex.com/docs/guides/faststore/organisms-hero):
```

> Sections are reusable across different pages. Developers determine which sections are available in the CMS. Below is an example of the [Hero Section](https://developers.vtex.com/docs/guides/faststore/organisms-hero):

Reviewer: I don't know if we should link the component here, since we are referring to the Hero section specifically.
